### PR TITLE
Creating 1.1.1

### LINKED ISSUE #63: Wrong version in README.md
##### SUMMARY

The `README.md` for `1.1.0` (and `main`) shows `1.0.0`, which is wrong.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

* digitalocean

##### EXPECTED RESULTS

The current version should be shown.

##### ACTUAL RESULTS

The previous version is shown.

### PR DESCRIPTION
**Pull request by mamercad**
##### SUMMARY

Creating `1.1.1` to fix the version in the `README.md`.

Fixes #63.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

* digitalocean

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ You can also include it in a `requirements.yml` file and install it via `ansible
 ---
 collections:
   - name: community.digitalocean
-    version: 1.0.0
+    version: 1.1.1
```
Comment (felixfontein) [inline README.md:64]: Because it's so easy to forget this one, I would simply remove the version here ;-)

**Comment by felixfontein** [comment]
galaxy.yml still says `version: 1.1.0`.

File: README.md (hunk 1 above)
Comment (gundalow) [reply README.md:64]: Yup, agreed remove this.

**Review by gundalow — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mamercad) [reply README.md:64]: Argh, forgot to stage `galaxy.yml`, and yes, I'll remove it from the `README.md`, good idea 😄

**Review by felixfontein — APPROVED** [review]

**Comment by mamercad** [comment]
> As a reviewer, it's strange to me that there is a fragment filename which cannot be found anywhere in history (or the commit log). Other than that, LGTM.

This might be on me -- seems like I never committed the fragment.

**Review by mpontillo — APPROVED** [review]

**Pull request merged**